src/main/java/com/laa/model/crime/CaseType.java

```diff
@@ -0,0 +1,7 @@
+package com.laa.model.crime;
+
+public enum CaseType {
+
+    MEG_COURT, CRIME_COURT
```

Comment:
Is this supposed to be MAG_COURT (magistrates)? and Should CRIME_COURT be CROWN_COURT?

Reply:
Thanks, will change it.